torch_xla/csrc/status.cpp

```diff
@@ -0,0 +1,78 @@
+#include "torch_xla/csrc/status.h"
+
+#include "absl/log/absl_check.h"
+#include "torch_xla/csrc/runtime/env_vars.h"
+#include "torch_xla/csrc/runtime/sys_util.h"
+
+namespace torch_xla {
+
+bool showCppErrorContext() {
+  return runtime::sys_util::GetEnvBool(runtime::env::kEnvShowCppErrorContext,
```

Comment:
This is called at every Status error construction site. It must be efficient. Use a local `static const bool` variable to remember the result so that we only call `GetEnvBool` once?